src/Compilers/CSharp/Portable/CommandLine/CSharpCommandLineParser.cs

```diff
@@ -1618,7 +1618,7 @@ private static IEnumerable<CommandLineAnalyzerReference> ParseAnalyzers(string a
                 yield break;
             }
 
-            List<string> paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path)).ToList();
+            IEnumerable<string> paths = ParseSeparatedPaths(value).Where((path) => !string.IsNullOrWhiteSpace(path));
```

Comment:
This is producing about 100 KB of garbage opening Roslyn. Every bit counts, don't buy the debugging argument that's what results view is for.